OpenTreeMap/src/OTM/Controllers/OTMMapViewController.m

```diff
@@ -49,10 +49,20 @@ @implementation OTMMapViewController
 
 @synthesize lastClickedTree, detailView, treeImage, dbh, species, address, detailsVisible, selectedPlot, mode, mapView, addTreeAnnotation, locationAnnotation, addTreeHelpView, addTreeHelpLabel, addTreePlacemark, locationActivityView, mapModeSegmentedControl, filters, filterStatusView, filterStatusLabel;
 
+- (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
+{
+    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
+    if (self) {
+        self.screenName = @"Map";  // for Google Analytics
```

Comment:
I'll delete that `initWithNibName` like the others. Initially I was setting the screen name there until I realized those methods were never called.